index.js

```diff
@@ -50,6 +65,10 @@ getUsersCreditDetails(usersArray);
 
 const genderView = users => {
   // Your code goes here ...
+  let femaleUsers = [], maleUsers = [];
+   users.filter(item =>  item.gender == 'female' ? femaleUsers.push(`${item.firstName} ${item.lastName}`) : maleUsers.push(`${item.firstName} ${item.lastName}`))
```

Comment:
this line does't really do what is intending to.
users.filter is not returning anything, but its pushing stuff, so its functioning more like a for each.
ideally you could have done this:
```js
let femaleUsers = users.filter(item => item.gender === "female")
let maleUsers = users.filter(item => item.gender === "male")
```

and here you would have the arrays already split by gender. and then you could do the transformation you want by mapping over them or something else.

The filter method you were doing, even though it probably did what you were expecting, it was not the goal of a filter. The filter must always return a value -> it must be assigned to a variable. Here its not only returning, (in the body of the function) and not being assigned anywhere